src/licensedcode/data/rules/afpl-8.0_url_4.RULE

```diff
@@ -0,0 +1,8 @@
+Please note that the program Triangle, 
```

Comment:
Are you sure that a `_url` suffix in the name is needed?

Reply:
No my mistake:( Changed it.